installation_and_upgrade/ibex_install_utils/tasks/system_tasks.py

```diff
@@ -402,25 +423,76 @@ def install_or_upgrade_git(self):
                 "Download and Install Git from https://git-scm.com/downloads"
             )
 
-    def confirm(self, message):
+    @task("Update visual studio redistributable files")
+    def install_or_upgrade_vc_redist(self) -> None:
+        """
+        Install the latest visual studio redistributable files
+        """
+
+        is_64_bit = platform.machine().endswith("64")
+        exe_file = Path(EPICS_CRTL_PATH, f"vc_redist.{'x64' if is_64_bit else 'x86'}.exe")
```

Comment:
It is the EPICS 32/64 bit that needs testing here, not the operating system (that will always be 64bit) - this is probably available via the arch argument that gets passed to the ibex deploy script. A 32bit epics needs both 32 and 64bit vc redistributable as there are both 32bit and 64 bit images on the computer